pkg/init/backend/interactive_test.go

```diff
@@ -0,0 +1,264 @@
+package backend
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	"github.com/devfile/library/pkg/devfile/parser"
+	parsercontext "github.com/devfile/library/pkg/devfile/parser/context"
+	"github.com/devfile/library/pkg/devfile/parser/data"
+	"github.com/devfile/library/pkg/testingutil/filesystem"
+
+	"github.com/redhat-developer/odo/pkg/catalog"
+	"github.com/redhat-developer/odo/pkg/init/asker"
+)
+
+func TestInteractiveBackend_SelectDevfile(t *testing.T) {
+	type fields struct {
+		buildAsker         func(ctrl *gomock.Controller) asker.Asker
+		buildCatalogClient func(ctrl *gomock.Controller) catalog.Client
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *DevfileLocation
+		wantErr bool
```

Comment:
This is not being used/set in any tests. Is that intentional?

Reply:
I think it's just from the template that your IDE provides for tests.